css/src/components/form/textarea.scss

```diff
@@ -87,3 +87,60 @@ $textarea-focus-box-shadow-size: 0 0.0625rem 0 0 !default;
 		}
 	}
 }
+
+$textarea-form-gap: 0.25em !default;
+$textarea-divider-radius: 0.5rem !default;
+
+.textarea-form {
+	display: grid;
+	gap: $textarea-form-gap;
+	padding-block: $control-padding-vertical;
+	padding-inline: $control-padding-horizontal;
+	border: $control-border-width solid transparent;
+	border-radius: $border-radius-lg;
+	border-color: $textarea-border-color;
+	background-color: $textarea-background-color;
+	font-size: $control-font-size;
+	border-block-end-color: $textarea-border-bottom-color;
+	outline-color: transparent;
+	transition: box-shadow 300ms linear;
+
+	@include prefers-reduced-motion {
+		animation-duration: 0s;
+	}
+
+	&:has(.textarea:focus-visible),
+	&:has(.textarea:hover) {
```

Comment:
also add `&:has(.textarea-form-footer:hover)` here? so the line stays blue even if the user hovers over the footer area

Reply:
This one is tricky, because the footer contains other focusable and hoverable elements, I'd lean toward not doing this one